### ISSUE TITLE
"nt" is not a good name for the generated library

### ISSUE THREAD

**Issue by ralphlange** [epics-base/normativeTypesCPP#2]
This tag introduces ambiguities and is misleading.

I thought this was a Windows configuration mixup when the pvAccess build did not find "libnt.a".
There is also a number theory library that uses the same name.

This will start creating serious trouble when making binary packages, as the package manager will complain, and Linux distributions do not like these name clashes at all.

Given that library names can use an arbitrary number of arbitrary characters, we should be able to find something more unique.

**Comment by dhickin** [comment]
+1

ntypes? (libntypes.a etc.)

**Comment by anjohnson** [comment]
`libnormative.a` seems a bit long, so yeah, `x_LIBS += ntypes` → `-lntypes` → `libntypes.a` seems good. The rename would obviously need changes in the Makefiles of all downstream modules, so would trigger a major release (4.6).

**Comment by gregoryraymondwhite** [comment]
Right, “nt” isn’t good. Shouldn’t it be simply “normativeTypes” - following
the convention the other modules already follow that main libname matches 
module name.

**Comment by dhickin** [comment]
If libnormative.a is a bit long then libnormativeTypes.a is definitely too long.

**Comment by gregoryraymondwhite** [comment]
Is this the 5 minute argument or full 1/2 hour?

First, would be an exception to the other modules - brain works on patterns. New users will want to find consistency. Second, “normative” is an adjective, whereas all the other modules’ libraries are sensibly named after their corresponding module noun.

nt is wrong
normative is silly
normativeTypes might be long, but is right as long as the module is named normativeTypesCPP.
